src/test/java/org/folio/login/it/LoginIT.java

```diff
@@ -140,6 +165,7 @@ void logout_positive_noCookieHeader() throws Exception {
       .andExpectAll(invalidatedCookie(testCookie2));
 
     assertThat(loadKcUserSessions()).isEmpty();
+    assertLogoutEvents(logoutAllEvent(USER_ID, KEYCLOAK_USER_ID));
```

Comment:
test calls `logout` endpoint but you expect logout-all event, do I miss something?

also don't see changes in the test for `logout-all` endpoints

Reply:
`noCookieHeader` means that we cannot recognize session id should be closed, then we should close all session related to the user